Server/Utilities/JSONUtils.m

```diff
@@ -37,11 +37,41 @@ + (NSArray *)elementTypes {
             sortedArrayUsingSelector:@selector(localizedCaseInsensitiveCompare:)];
 }
 
++ (CBXVisibilityResult *)visibilityResult: (XCUIHitPointResult*)hitPoint {
+    CBXVisibilityResult *cbxResult = [CBXVisibilityResult new];
+    CGPoint intermediate;
+    XCUIHitPointResult *result = hitPoint;
+
+    cbxResult.isVisible = result.isHittable;
+
+    if (result.isHittable) {
+        intermediate = result.hitPoint;
+    } else {
+        intermediate = CGPointMake(-1, -1);
+    }
+
+    cbxResult.point = intermediate;
+
+    DDLogDebug(@"Finding hit point for XCElementSnapshot: %@\n"
+               "hitpoint: %@\n"
+               "visible: %@\n",
+               self,
+               [NSString stringWithFormat:@"(%@, %@)", @(intermediate.x), @(intermediate.y)],
+               result.isHittable ? @"YES" : @"NO");
+
+    return cbxResult;
+}
+
 + (NSDictionary *)snapshotOrElementToJSON:(id)element {
     NSMutableDictionary *json = [NSMutableDictionary dictionary];
 
+    
     XCElementSnapshot *snapshot;
-    if ([element isKindOfClass:[XCElementSnapshot class]]) {
+    
+    //In Xcode 14 [XCElementSnapshot class] is not available, so this is the only way to compare classes.
+    Class targetClass = NSClassFromString(@"XCElementSnapshot");
```

Comment:
Did you looked for XCElementSnapshot in other frameworks? Sometimes it can be just moved.

Reply:
It wasn't moved, but its linking now doesn't work

Reply:
Ok